Cargo.toml

```diff
@@ -34,6 +35,7 @@ mysql-rustls = ["sqlx/mysql", "sqlx/tls-rustls"]
 mysql-native = ["sqlx/mysql", "sqlx/tls-native-tls"]
 redis-db = ["redis_pool", "redis"]
 redis-clusterdb = ["redis_pool/cluster", "redis/cluster-async"]
+session-any-pool = []
```

Comment:
feature is good here but we also need to add it to the [package.metadata.docs.rs] within cargo.toml so it shows up in the docs and add the feature with a short description to the feature list within readme.

Reply:
added